web/client/utils/FeatureInfoUtils.js

```diff
@@ -77,7 +77,7 @@ const Validator = {
          *Parse the JSON to get only the valid json responses
          */
         getValidResponses(responses) {
-            return responses.filter((res) => res.response && res.response.features && res.response.features.length);
```

Comment:
you did this change because also response with no features should be considered valid?

Reply:
That's correct